tests/integration/pilot/common/traffic.go

```diff
@@ -285,7 +285,7 @@ func RunAllTrafficTests(t framework.TestContext, i istio.Instance, apps deployme
 	RunSkipAmbient("use-client-protocol", useClientProtocolCases, "https://github.com/istio/istio/issues/43161")
 	RunCase("destinationrule", destinationRuleCases)
 	RunCase("vm", VMTestCases(apps.VM))
-	RunCase("dns", DNSTestCases)
+	RunSkipAmbient("dns", DNSTestCases, "https://github.com/istio/istio/issues/45923")
```

Comment:
LGTM, but how much coverage do we gain with this skipped?

Reply:
I _thought_ this test was passing with the mode where we make everything get a Waypoint by default. Won't block PRs on that until CI blocks on that. 

https://github.com/istio/istio/blob/e430c81abbeba1cfb03caeff882eb4ad4ff33094/pkg/test/framework/resource/flags.go#L184-L185